### ISSUE TITLE
Node version breaking change

### ISSUE THREAD

**Issue by jdanil** [survivejs/webpack-merge#138]
Releases from 5.0.0-beta-1 include a breaking change to the required node version that isn't documented in the changelog. The `package.json` engines field was updated to include `"node": ">=12.0.0"`, where previously an engine wasn't specified.

Running install with node 10 returns the following error.

```
error webpack-merge@5.0.7: The engine "node" is incompatible with this module. Expected version ">=12.0.0". Got "10.21.0"
error Found incompatible module.
```

Considering that this is an undocumented breaking change and node 10 is still supported until April 2021 (https://nodejs.org/en/about/releases/) I'm wondering if this was a mistake?

**Comment by bebraw** [comment]
Thanks for the heads up! I will add support for Node 10 as that’s good to have still.

**Comment by bebraw** [comment]
I included support for Node 10 to `5.0.8` and added the missing changelog bits. Thanks again!

**Comment by jdanil** [comment]
Thank you!
